documentation/README.md

```diff
@@ -0,0 +1,81 @@
+# H2O LLM Studio Documentation
+
+- The LLM Studio documentation is built using [Makersaurus](https://h2oai.github.io/makersaurus/) which is a very thin wrapper around Facebook's Docusaurus.
+- The documentation is displayed at {{ https://docs.h2o.ai/h2o-llm-studio/ }}
+
+To view, edit, and cut a version of the documentation, the following is required:
+
+- Node.js version 16.14+ (you can check your version by running `node -v`). Use nvm to manage multiple Node versions installed on a single machine.
+
+- To install Node.js and npm with nvm in Mac or Ubuntu, run: `curl -o-
+https://raw.githubusercontent.com/creationix/nvm/v0.33.0/install.sh | bash` and `nvm install node`
+
+### Documentation structure
+
+
+```
+├── documentation
+│ ├── docs
+│ ├── tmp
+│ ├── makersaurus.config.js
+│ ├── sidebars.js
+│ ├── package.json
+│ ├── package-lock.json
+```
+
+- `documentation/docs`: Contains Markdown documentation files to edit the next documentation version.
+Customize the order of the docs sidebar in `sidebars.js`
+- `documentation/tmp`: Temporary files generated by Makersaurus. Do not edit these files.
+- `documentation/makersaurus.config.js`: Makersaurus [config file](https://h2oai.github.io/makersaurus/api/config)
+- `documentation/sidebars.js`: Sidebar configuration file
+- `documentation/package.json`: npm configuration file
+- `documentation/package-lock.json`: Generated by npm. Do not edit this file.
+
+
+### Edit locally
+
+To setup the local `env` to view and edit the next or past documentation versions ([first, ensure you install
+Node.js](#requirements)):
+
+1. Enter the documentation folder
+
+`cd documentation`
+
+2. Install dependencies
+
+`npm install`
```

Comment:
Getting an error 401 here
`npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@h2oai/makersaurus/0.4.0/b35eed6758ef47d5a203d8e9c909d3dff723b2ec - authentication token not provided`

Reply:
Hi @pascal-pfeiffer, this can be resolved by following the steps @oshi98 just added to the readme with [this commit](https://github.com/h2oai/h2o-llmstudio/pull/156/commits/5674fa627063e4238fdf9f2dc0207d2a1abe866a).

Reply:
Does this work for non-h2o accounts?

Reply:
It should work. The Makersaurus npm package is public. But we haven't explicitly tested it out with a non-h2o account.

Reply:
Update: I tested and verified this with a brand new non-h2o github account and it worked.